pakyow-core/lib/pakyow/core/routing/route.rb

```diff
@@ -59,19 +59,6 @@ def populated_path(path_to_self, **params)
         }.join("/"))
       end
 
-      def freeze
-        hooks.each do |_, hooks_arr|
```

Comment:
I believe we also need to refine `Hash` to freeze values. Let me know if I'm missing something.